parachain/primitives/beacon/src/lib.rs

```diff
@@ -5,9 +5,11 @@
 pub mod bits;
 pub mod bls;
 pub mod config;
+pub mod deneb;
 pub mod merkle_proof;
 pub mod receipt;
 pub mod ssz;
+pub mod ssz_deneb;
```

Comment:
Ditto, this module layout can be improved.

For example:
```rust
use crate::ssz::capella::SSZExecutionPayloadHeader
use crate::types::deneb::SSZExecutionPayloadHeader
```

Reply:
https://github.com/Snowfork/snowbridge/pull/981/commits/3580c4de7d6400880f3a4df171ffb42b5000737b

Though `use crate::ssz::deneb::SSZExecutionPayloadHeader` instead because there are some conflict imports between `ssz_rs` and `sp_core` which causes a bit messy to merge.